kod/object/active/holder/nomoveon/battler/monster/towns/tostwn/tswatch.kod

```diff
@@ -2019,6 +2057,32 @@ messages:
 
    ReqSomethingAttack(what=$,victim=$)
    {
+      % if a fight isnt in session, attack fails.
```

Comment:
There isn't a watcher class and I have no idea where else to put this shared code snippet.

This type of logic used to live in Room.kod but it wasn't implemented correctly as the Monster check never occurred because the attacker failed an IsCombatant check.

It feels really icky to leave this code in Room but I don't know a way to avoid some repetition here between Tos Watcher and Goad.  In principle the two watchers could have different logic for what sort of attacks are allowed - although I don't have a good example of why we might want this.